### ISSUE TITLE
Please make a firmware for ENDER 5 plus with E3 3.0

### ISSUE THREAD

**Issue by Propeople2** [bigtreetech/BIGTREETECH-SKR-mini-E3#674]
Pleaseeeeeeeee!!!!!

**Comment by Scoutprime** [comment]
Hi, same request :) i'm trying to create firmware for E5plus with CR10 stock display and BLtouch. So far, no luck. Still having issues with Z Axis. It would be great if BigTreeTech will create Marlin for E5plus

**Comment by mcxpress80** [comment]
yes it would be nice

**Comment by Pseudo-Nym** [comment]
Agree

**Comment by Dazza2012** [comment]
I'm having issues with Bed Levelling stopping at different times during mesh setup attempts using this recently purchased SKR E3 MINI v3.0 board and BTT TFT 35 v3.0.1 screen. BigTreeTech states this board is suitable for Ender 5 Plus but there is no firmware available.
I cannot find any suitable Marlin Firmware or Config files on BigTreeTecch GitHub so far. 
Firmware files for this would be good including BL Touch options. Marlin Config file samples to work with suited for Ender 5 Plus is what I am really after! 
Help!
Thanks.

**Comment by Gixie69** [comment]
Yeah for sure, We need some firmware for the E5+ Skr mini3 v3.0.1 Ive had to use TH3D firmware but its still not having it. Wont home, wont level Will try to print but the extruders not working. Im guessing its all in Marlin but im a trucker not a computer wizz. Real people need helping out BTT.
